etc/profile-a-l/lutris.profile

```diff
@@ -70,5 +74,6 @@ shell none
 #private-dev
 private-tmp
 
-dbus-user none
-dbus-system none
+# Games with enabled Feral GameMode won't run if D-Bus system is disabled.
+#dbus-user none
+#dbus-system none
```

Comment:
Does `dbus-system none` really break gamemode?

Reply:
You're right. `dbus-system none` doesn't break GameMode.

Reply:
I've updated commit.